x-pack/plugin/searchable-snapshots/src/test/java/org/elasticsearch/xpack/searchablesnapshots/AbstractSearchableSnapshotsRestTestCase.java

```diff
@@ -61,7 +64,7 @@ private void runSearchableSnapshotsTest(SearchableSnapshotsTestCaseBody testCase
 
         logger.info("creating index [{}]", indexName);
         createIndex(indexName, Settings.builder()
-            .put(IndexMetaData.SETTING_NUMBER_OF_SHARDS, numberOfShards)
+            .put(SETTING_NUMBER_OF_SHARDS, numberOfShards)
```

Comment:
nit: seems unnecessary,  and we use the fully-qualified  name below.

Reply:
I reverted this change in 244b89e1278f52e5ff2d1f7b3b550b7cafaf56f6